spec/export_spec.rb

```diff
@@ -15,6 +15,11 @@
       File.read mock_file
     }
 
+    let (:mock_data_closed) {
+      mock_file = File.expand_path('../fixtures/issues_closed.json', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.